Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -1,29 +1,37 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
-version: 2
 name: MultiMarkdown
-first_line_match: (?i)^format:\s*complete\s*$
 scope: text.html.markdown.multimarkdown
+
+extends: Packages/Markdown/Markdown.sublime-syntax
+
+first_line_match: (?i:^format:\s*complete\s*$)
+
 variables:
-  header: ((?=[A-Za-z0-9])[\w -]+)(:)
+  header: ([A-Za-z0-9][\w -]*)(:)
```

Comment:
Not sure why you use capturing groups in variables here. I thought convention was to not use those in variables but where they're used instead for better readability.